src/main.rs

```diff
@@ -135,24 +135,31 @@ fn main() -> Result<()> {
                 .help(&format!(
                     "Specifies a preferred color set. Unspecified colors will remain as default.
 Possible values: [{0}{1}{2}{3}{4}{5}{6}{7}{8}{9}{10}{11}{12}{13}{14}{15}]",
-                    "0".black(),
-                    "1".red(),
-                    "2".green(),
-                    "3".yellow(),
-                    "4".blue(),
-                    "5".magenta(),
-                    "6".cyan(),
-                    "7".white(),
-                    "8".bright_black(),
-                    "9".bright_red(),
-                    "10".bright_green(),
-                    "11".bright_yellow(),
-                    "12".bright_blue(),
-                    "13".bright_magenta(),
-                    "14".bright_cyan(),
-                    "15".bright_white(),
-                )),
-        )
+                "0".black(),
+                "1".red(),
+                "2".green(),
+                "3".yellow(),
+                "4".blue(),
+                "5".magenta(),
+                "6".cyan(),
+                "7".white(),
+                "8".bright_black(),
+                "9".bright_red(),
+                "10".bright_green(),
+                "11".bright_yellow(),
+                "12".bright_blue(),
+                "13".bright_magenta(),
+                "14".bright_cyan(),
+                "15".bright_white(),
+            )))
+        .arg(
+            Arg::with_name("no-bold")
+                .short("b")
+                .long("no-bold")
+                .help(&format!(
+                    "{}",
+                    "Turns off bold formatting for the logo and all labels"
+            )))
```

Comment:
You don't need to use `format!` here, since you aren't parsing values into your string. `help` takes type `&str`, which a string literal already is without being formatted.
Basically, this results in `"Turns off bold formatting for the logo and all labels"` going through the types `&str` -> `String` -> `&str`.

```rust
help("Turns off bold formatting for the logo and all labels") // <- this should be fine
```